src/olympia/devhub/views.py

```diff
@@ -1385,7 +1383,7 @@ def decorator(f):
         @functools.wraps(f)
         def wrapper(request, *args, **kw):
             step = outer_step
-            max_step = 6
+            max_step = 5
```

Comment:
I had hoped you would have removed this. Now I understand why you kept the ugly URL names... Can you simplify this and just make an ordered list of url names ? Then you can figure out which one to go to next by looking at the next in the list ?